tsconfig.json

```diff
@@ -13,7 +13,11 @@
     "dist/**/*",
     "release/**/*",
     "webpack.main.ext.js",
+<<<<<<< HEAD
```

Comment:
can you clean up this merge conflict

Reply:
just pushed it. I removed tests from exclude so we still get type checking in our tests